app/src/main/java/org/mifos/mobilebanking/ui/fragments/RegistrationFragment.java

```diff
@@ -101,7 +101,7 @@ public void registerClicked() {
                 payload.setPassword(etPassword.getText().toString());
             }
             payload.setPassword(etPassword.getText().toString());
-            payload.setUsername(etUsername.getText().toString());
+            payload.setUsername(etUsername.getText().toString().trim());
```

Comment:
@therajanmaurya This line will execute when there is no space in between, that's why I have used string.trim() here.